File: java/Mdc.java
public class Mdc {

    public static int mdc(int a, int b) {
        while (b != 0) {
            int resto = a % b;
            a = b;
            b = resto;
        }
        return a;
    }

    public static void main(String[] args) {
        System.out.println("--- MDC ---");
        System.out.println("MDC(48, 18) = " + mdc(48, 18));
    }
}

File: javascript/mdc.js
function mdc(a, b) {
  while (b !== 0) {
    const resto = a % b;
    a = b;
    b = resto;
  }
  return a;
}

console.log("--- MDC ---");
console.log(`MDC(48, 18) = ${mdc(48, 18)}`);
